src/ast.rs

```diff
@@ -5,18 +5,24 @@ pub struct Call {
 }
 
 #[derive(Debug)]
-pub enum UnaryPlus {
-    Integer(i32),
-    FloatingPoint(f64),
-    Identifier(String),
+pub enum PlusOrMinus {
```

Comment:
I wonder if this should be `pub enum Operation` (possibly `UnaryOperation`, but that precludes its reuse for binary operators...